MaxText/train.py

```diff
@@ -494,9 +500,21 @@ def main(argv: Sequence[str]) -> None:
       stack_trace_to_cloud = config.stack_trace_to_cloud,
       stack_trace_interval_seconds = config.stack_trace_interval_seconds))
   diagnostic_config = diagnostic_configuration.DiagnosticConfig(debug_config)
+  logger_name = f'goodput_{config.run_name}'
```

Comment:
This is duplicate of lines 420 and 421.

Reply:
Discussed offline. Keeping two instantiations of GoodputRecorder to avoid conditional logic when passing the object to train_loop.